src/components/Filters/Filters.jsx

```diff
@@ -0,0 +1,132 @@
+import cn from 'classnames';
+
+export const Filters = ({
+  users,
+  selectedUser,
+  changeSelectedUser,
+  searchQuery,
+  changeSearchQuery,
+  categories,
+  selectedCategories,
+  changeCategory,
+  clearCategories,
+}) => {
+  const resetAllFilters = () => {
+    changeSelectedUser(0);
+    changeSearchQuery('');
+    clearCategories();
+  };
```

Comment:
as this function works with App state, will be better to move it to the `App` component and pass as props